Include/cpython/import.h

```diff
@@ -32,7 +32,7 @@ struct _frozen {
     const char *name;                 /* ASCII encoded string */
     const unsigned char *code;
     int size;
-    PyObject *(*get_code)(void);
+    PyObject *(*get_code)(void);      /* Returns strong ref */
```

Comment:
The comment is obsolete. Strong refs are default.

Reply:
@vstinner asked for this https://github.com/python/cpython/pull/30987#pullrequestreview-866046549 also even if strong refs are default a little extra information doesn't hurt.

Reply:
IMHO it's distracting noise. Default behavior does not need a comment.

Reply:
The whole deepfreeze system is currently very least documented so this helps without going through the whole code that it returns a strong ref.

Reply:
Every function returning a ``PyObject *`` returns a strong reference **unless** it's documented otherwise. There is no point in documenting that a function returns a strong reference. Comments should be helpful and reserved for cases that are not obvious or that behave differently.

Reply:
Removed